instagram/src/Comments.js

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import './App.css';
+
+export default class Comments extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: {}
```

Comment:
If this is going to become an array of comments, you should initialize it to an array.